asreview/webapp/api/projects.py

```diff
@@ -1166,6 +1166,10 @@ def api_export_dataset(project):
         # read the dataset into a ASReview data object
         as_data = read_data(project)
 
+        # Add a new column 'is_prior' to the dataset
+        state_df["asreview_prior"] = state_df.query_strategy.eq("prior").astype(int)
+        as_data.df = as_data.df.drop("asreview_prior", axis=1).join(state_df["asreview_prior"].astype(int), on="record_id")
+
```

Comment:
We can keep this and revert the changes to ris_reader.py and ris_writer.py. Then it should work. 

I don't know what to do with this info during the setup yet. Currently, it's ignored in favor of the labeled column.